doc/sfp-cmis/Interface-Link-bring-up-sequence.md

```diff
@@ -184,18 +187,272 @@ if transceiver is not present:
+    OA ->> APPL_DB : DEL PORT_TABLE
+
+    XCVRDMT -x APPL_DB : XCVRD main thread proecesses DEL event of APPL_DB PORT_TABLE
+    Note over XCVRDMT: generate_sigabrt = True
+    alt If threads > 0 are dead
+        XCVRDMT -x XCVRDMT : Kill XCVRD with SIGKILL
+    end
+    XCVRDMT -x CmisManagerTask : Stop CmisManagerTask
+    deactivate CmisManagerTask
+    XCVRDMT -x DomInfoUpdateTask : Stop DomInfoUpdateTask
+    deactivate DomInfoUpdateTask
+    XCVRDMT -x SfpStateUpdateTask : Stop SfpStateUpdateTask
+    deactivate SfpStateUpdateTask
+    Note over XCVRDMT : deinit()
+    alt self.sfp_error_event.is_set()
+        XCVRDMT -x XCVRDMT : sys.exit(SFP_SYSTEM_ERROR)
+    else if generate_sigabrt is True
+        XCVRDMT -x XCVRDMT : Kill XCVRD with SIGABRT
+
+    else
+        XCVRDMT -x XCVRDMT : Graceful exit
+    end
+    deactivate XCVRDMT
+```
+
+## Test plan and expectation
+| Event          | APPL_DB_<asic_n> cleared | Xcvrd restarted | NPU SI settings renotify | NPU_SI_SETTINGS_SYNC_STATUS value on xcvrd boot-up for initialized transceiver | CMIS re-init triggered | Link flap |
+| -------------- | ------------------------ | --------------- | ------------------------ | ------------------------------------------------------------------------------ | ---------------------- | --------- |
+| Xcvrd restart  | N                        | Y               | N                        | NPU_SI_SETTINGS_DONE                                                           | N                      | N         |
+| Pmon restart   | N                        | Y               | N                        | NPU_SI_SETTINGS_DONE                                                           | N                      | N         |
+| Swss restart   | Y                        | Y               | Y                        | NPU_SI_SETTINGS_DEFAULT                                                        | Y                      | N/A       |
+| Syncd restart  | Y                        | Y               | Y                        | NPU_SI_SETTINGS_DEFAULT                                                        | Y                      | N/A       |
+| config reload  | Y                        | Y               | Y                        | NPU_SI_SETTINGS_DEFAULT                                                        | Y                      | N/A       |
+| Cold reboot    | Y                        | Y               | Y                        | NPU_SI_SETTINGS_DEFAULT                                                        | Y                      | N/A       |
+| Config shut    | N                        | N               | N                        | NPU_SI_SETTINGS_DONE                                                           | N                      | N/A       |
+| Config no shut | N                        | N               | N                        | NPU_SI_SETTINGS_DONE                                                           | N                      | N/A       |
```

Comment:
Correct 'config shut' and 'config no shut' events to 'config interface shut' and 'config interface startup'

Reply:
I have addressed this now.